app/decorators/activity_decorator.rb

```diff
@@ -2,9 +2,9 @@ module ActivityDecorator
   include CommonDecorator
 
   def summary_html
-    summary =  "#{ created_at_string } | " +
+    summary =  "<small>#{ created_at_string } | " +
       "#{ link_to(user.nickname, user_path(user)) }さんは" +
-      "「#{ link_to(notice.title, notice_path(notice)) }」" +
+      "「#{ link_to(truncate(notice.title, length: 15), notice_path(notice)) }」" +
```

Comment:
Line is too long. [81/80]<br>Align the operands of an expression in an assignment spanning multiple lines.